packages/api/src/schema/api-schema.yaml

```diff
@@ -1483,6 +1463,20 @@ components:
                       description: |
                         URL to access metadata file via HTTP through an IPFS
                         gateway
+            exportData:
```

Comment:
Even though our API schema itself is mostly in `camelCase`, I think the components in the OpenAPI specs are in `kebab-case`. So this should be `export-data` instead (and `storage-status`, etc). 

This is way less important than the actual user-facing fields tho, so don't bother if it's a pain to rename it now

Reply:
Renamed `storage-status`. But I'll keep `exportData` since it's not a component, just a key/field. And we use camelCase for other cases like it.

Reply:
I see, good to know. Yeah we def need to fix this schema further. It actually gets a bunch of errors if you feed it to some OpenAPI validator and we probably need to setup some linter or sth if we want to guarantee any real consistency.